src/read.rs

```diff
@@ -457,14 +457,14 @@ impl<R: Read + io::Seek> ZipArchive<R> {
     }
 
     /// Get a contained file by index
-    pub fn by_index<'a>(&'a mut self, file_number: usize) -> ZipResult<ZipFile<'a>> {
+    pub fn by_index(&mut self, file_number: usize) -> ZipResult<ZipFile> {
```

Comment:
Can you add the elided lifetime for `ZipFile<'_>`? The borrow is an important part of the signature and iirc, it affects the docs.

Reply:
Done